Recipient View and Component Changes

jacksteiner opened the pull request:
created contract preview component which is now in contract details and recipientview. Email is sent after send to recipient button is clicked but the link in the email is not working yet.

If this should have been a new branch or this is to confusing to merge I'd be willing to hop on a call to help or I can just create a new branch and pull request with only the new components and respective name changes in them.

jacksteiner committed:
created contract preview component which is now in contract details and recipientview. Email is sent after send to recipient button is clicked but the link in the email is not working yet

jdh2021 commented:
Hi Jack, would you be able to split this out by feature? I agree there's a lot of changes affecting different components and routes. I think it would be easier to piece together and avoid conflicts.  Just a few comments/questions as well:

1. It looks like the bulk of ContractDetails was moved into a ContractPreview component. I know we had talked about a stretch being implementing a Preview button that would display the full legal contract so I wasn't clear if that's what this component is related to?
2. I was working on the user router POST today  (#44 on task board) and aim to have the insert into user and user_contract completed by tomorrow so you won't need to submit changes to user.router.js.
3. Will you be adding Accept and Decline buttons to RecipientView? I think we'll need these to set the contract key in the reducer and update the contract status if the recipient declines.

oscar-oscar committed:
Merge branch 'main' into feature-recipientroutersendgrid

oscar-oscar commented:
ContractPreview is displaying the card, not a PDF preview.

oscar-oscar committed:
Update user.router.js

reverted Register to avoid conflicts with Jennifer's code. Worked with Chris on this.

oscar-oscar commented:
Worked with Chris to resolve merge conflicts. Reverted the Register router to avoid conflicts with Jennifer's code
